src/components/InteractiveTutorial/MdxPages/SparseVectors.mdx

```diff
@@ -0,0 +1,148 @@
+export const title = 'Sparse Vectors';
+
+# Sparse Vector Search
+
+In this tutorial, you'll learn how to create a collection with sparse vectors in Qdrant, insert points with sparse vectors, and query them based on specific indices and values. Sparse vectors allow you to store and query vectors where only certain dimensions have non-zero values, making them efficient for applications like text embeddings or sparse data.
+
+## Step 1: Create a collection with sparse vectors
+
+The first step is to create a collection that can handle sparse vectors. In this example, we’ll create a collection called `sparse_charts` with a vector size of 4 and the Cosine distance metric for similarity calculation. The sparse vectors will be indexed in memory (no on-disk storage).
+
+Run the following request to create the collection:
+
+```json withRunButton=true
+PUT /collections/sparse_charts
+{
+    "vectors": {
+        "size": 4,  
+        "distance": "Cosine"  
+    },
+    "sparse_vectors": {
+        "text": {  
+            "index": {
+                "on_disk": false  
+            }
+        }
+    }
+}
```

Comment:
```suggestion
PUT /collections/sparse_charts
{
    "sparse_vectors": {
        "text": {  
            "index": {
                "on_disk": false  
            }
        }
    }
}
```

It isn't required to pass dense vectors configuration if you plan to use only sparse vectors.